test/server.test.js

```diff
@@ -145,5 +156,16 @@ test(`POST and GET location for \`mcs\` route`, async t => {
     url: '/route/mcs/location'
   })
   t.equal(get_response.statusCode, 200, 'GET returns a status code of 200')
-  t.equal(post_response.body, JSON.stringify(location), "GET returns a body equal to POST'ed coordinates")
```

Comment:
This is a bug in the tests that I'll make a separate PR for, to be merged before this one